addons/web/static/src/views/pivot/pivot_controller.xml

```diff
@@ -42,7 +42,7 @@
                     </t>
                     <t t-else="" t-call="web.NoContentHelper"/>
                 </t>
-                <t t-component="props.Renderer" model="model"/>
+                <t t-component="props.Renderer" model="model" buttonTemplate="props.buttonTemplate"/>
```

Comment:
buttonTemplate should still be in the props definition of PivotController, but optional. With the current version, `props.buttonTemplate` can never be anything else than undefined, as `buttonTemplate` isn't a valid props